pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -101,6 +101,25 @@ def test_to_datetime_format_YYYYMMDD(self, cache):
         expected = Series(["20121231", "20141231", "NaT"], dtype="M8[ns]")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "input_s, expected",
+        [
+            # strings with None
+            [
+                Series(["19801222", None, "20010112"]),
+                Series([Timestamp("19801222"), Timestamp(None), Timestamp("20010112")]),
+            ],
+            # Integers with None
+            [
+                Series([19801222, None, 20010112]),
+                Series([Timestamp("19801222"), Timestamp(None), Timestamp("20010112")]),
+            ],
+        ],
+    )
+    def test_to_datetime_format_YYYYMMDD_with_none(self, input_s, expected):
+        result = pd.to_datetime(input_s, format="%Y%m%d")
```

Comment:
can you add the issue number as a comment

Reply:
Added the issue the number as comment